packages/cacti-plugin-copm-corda/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@hyperledger-cacti/cacti-plugin-copm-corda",
+  "version": "2.0.0",
+  "description": "Cacti Common Operators - Corda",
+  "homepage": "https://github.com/hyperledger-cacti/cacti#readme",
+  "bugs": {
+    "url": "https://github.com/hyperledger-cacti/cacti/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger-cacti/cacti.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Jennifer Bell",
+  "main": "dist/lib/main/typescript/index.js",
+  "module": "dist/lib/main/typescript/index.js",
+  "types": "dist/lib/main/typescript/index.d.ts",
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "build": "./gradlew build",
+    "bundleopenapi": "yarn run --top-level tools:bundle-open-api-tpl-files"
+  },
+  "dependencies": {
+    "@bufbuild/buf": "1.30.0",
+    "@bufbuild/protobuf": "1.10.0",
+    "@connectrpc/connect": "1.4.0",
+    "@connectrpc/connect-node": "1.4.0",
+    "@connectrpc/connect-web": "1.4.0",
+    "@hyperledger-cacti/cacti-copm-core": "2.0.0",
+    "@hyperledger-cacti/cacti-copm-test": "2.0.0",
+    "@hyperledger/cacti-weaver-protos-js": "2.0.0",
+    "@hyperledger/cacti-weaver-sdk-fabric": "2.0.0",
```

Comment:
`2.0.0` version doesn't exist for the two packages above. Because of the organization change, the last version tagged under `@hyperledger` was `[2.0.0-rc.7](https://github.com/orgs/hyperledger/packages/container/cacti-weaver-driver-fabric/283162801?tag=2.0.0-rc.7)`. Weaver publish actions failed during the `2.0.0` release because paths were not updated. These actions should succeed when the next release (`2.0.1`) is done, hopefully in the next few days. So you have two options: (1) wait for the next release and then update these version numbers here and elsewhere in your code, or (2) use the last release version `2.0.0-rc.7`.

Reply:
These refer to the local monorepo versions, which is why they are able to compile.  The packages are built in the cactus way, with `npm run configure` at the top level.  This uses the tsconfig file at at the root level which builds the packages in a specific order and keeps a reference of local packages.  One of the challenges of this project was getting everything to build successfully given the differences in build philosophy between cactus and weaver.

Reply:
OK, I guess this works for a completely local build, i.e., using nothing outside the source code. It's not that the Weaver build build philosophy was different, but that we want the option to publish reusable assets (packages, containers) and then allow clients and third-party apps to depend on them rather than have to build everything locally. So we used actions to ensure that the flows worked both with locally-built code and published packages.

Anyway, this works for now. The Cacti maintainers can have a discussion later about building packages with external dependencies.